src/Tribe/Admin/Settings.php

```diff
@@ -406,7 +407,17 @@ public function sort_tabs( $tabs, $admin_page ): array {
 
 		// Ensure these are the first tabs.
 		$first   = [ 'general', 'display' ];
-		$tabs   = array_merge( array_flip( $first ), $tabs );
+
+		// Reverse to maintain order when prepending
+		foreach ( array_reverse( $first ) as $sort ) {
```

Comment:
Not a major thing when there are only 2 but `array_reverse()` (or any callback in the loop args) will run for _each and every iteration_ - so it's better to do something like:
```suggestion
		$reversed_arr = array_reverse( $first);
		
		foreach ( $reversed_arr as $sort ) {
```